### ISSUE TITLE
[0.7] Example `server_fns_axum` crashes

### ISSUE THREAD

**Issue by sectore** [leptos-rs/leptos#3055]
**Describe the bug**

Example `server_fns_axum` crashes

**To Reproduce**

Checkout latest `main`.

```sh
cd /examples/server_fns_axum
cargo make ci
cargo make start
```
Open browser http://127.0.0.1:3000 and check Console:

```
panicked at leptos/tachys/src/view/strings.rs:94:14:
couldn't cast text node from node


server_fns_axum.js:440:9
```

**Comment by gbj** [comment]
Oh thanks I just opened #3056 describing the same thing which I'm pretty sure is the cause. Could you revert the commit that fixes it by adding HTML entities? It's supposed to work without them and entities aren't supported client-side in text nodes so will cause other trouble in an example.

**Comment by sectore** [comment]
Ok. Reverted 5b41411.

**Comment by sectore** [comment]
Duplicated by #3056.
Fixed by 6df8657.
